packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -209,6 +209,21 @@ describe('network stubbing', { retries: 2 }, function () {
       cy.wait('@create')
     })
 
+    // https://github.com/cypress-io/cypress/issues/9313
+    it('lower-cased method works', () => {
+      cy.intercept('post', 'http://dummy.restapiexample.com/api/v1/create').as('create')
```

Comment:
in the future, we should try to use the internal server for tests only, not 3rd party - updated this + the earlier test